scripts/check-pkg-for-release.js

```diff
@@ -44,13 +44,16 @@ const MASTER_PKG = require(path.join(process.env.PWD, MASTER_PKG_PATH));
 // directly included in webpack or the material-component-web module. But they
 // are necessary since other MDC packages depend on them.
 const CSS_WHITELIST = ['base', 'animation', 'auto-init', 'rtl', 'selection-control'];
+const NOT_PUBLISHABLE = ['shape'];
 
 main();
 
 function main() {
   checkPublicConfigForNewComponent();
   if (pkg.name !== MASTER_PKG.name) {
     checkNameIsPresentInAllowedScope();
+  }
+  if (NOT_PUBLISHABLE.indexOf(pkg.name) !== -1) {
```

Comment:
Uh... is this inverted? IIUC this *will only* execute the statements below if the package is *not* publishable.

Reply:
oops, fixed this up